app/controllers/Api.scala

```diff
@@ -282,6 +286,12 @@ object Api extends Controller with PanDomainAuthActions with SharedSecretAuth {
     }
   }
 
+  def priorities = CORSable(atomCorsAble) {
+    AuthAction {
```

Comment:
Should this be an `APIAuthAction`?

Reply:
Agreed, all the metadata endpoints should be (`statuses`, `sections`, `allowedAtomTypes` and `priorities`).

I've raised https://github.com/guardian/workflow-frontend/pull/169 for it